torch/_dynamo/resume_execution.py

```diff
@@ -531,10 +531,12 @@ def remap_block_offsets(
                     for inst, o in zip(
                         prefix_blocks, meta.prefix_block_target_offset_remap
                     ):
-                        meta.block_target_offset_remap[inst.offset] = o
+                        block_target_offset_remap[cast(int, inst.offset)] = o
 
                     # old bytecode targets are after the prefix PUSH_EXC_INFO's
-                    old_start_offset = prefix_blocks[-1].offset if prefix_blocks else -1
+                    old_start_offset = (
+                        cast(int, prefix_blocks[-1].offset) if prefix_blocks else -1
```

Comment:
should this be a cast or just an `int()` call?

Reply:
It appears to already be an `int` at runtime. The `cast()` call (which has no runtime effect) is just there to cast `Optional[int]` into `int`.

Reply:
in that case, maybe better to `assert isinstance(prefix_blocks[-1].offset, int)`